### PR TITLE
User authentication completed.

### PR DESCRIPTION
**Pull request by JakeWengroff**
Helloooooo @Brit200313!  I didn't want to bother, so I held off for a few days.
Thanks for reviewing.
Jake

### CODE REVIEW

File: app/assets/javascripts/welcome.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (Brit200313) [inline app/assets/javascripts/welcome.coffee]: Let's delete this! Not using it.

File: app/assets/javascripts/welcome.coffee [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```
Comment (Brit200313) [inline app/assets/javascripts/welcome.coffee]: Delete please!

File: app/assets/stylesheets/welcome.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the welcome controller here.
```
Comment (Brit200313) [inline app/assets/stylesheets/welcome.scss]: Delete please!

File: app/controllers/welcome_controller.rb [hunk 4]
```diff
@@ -0,0 +1,4 @@
+class WelcomeController < ApplicationController
```
Comment (Brit200313) [inline app/controllers/welcome_controller.rb]: Delete please!

File: app/helpers/welcome_helper.rb [hunk 5]
```diff
@@ -0,0 +1,2 @@
+module WelcomeHelper
```
Comment (Brit200313) [inline app/helpers/welcome_helper.rb]: Delete please!

File: app/views/welcome/about.html.erb [hunk 6]
```diff
@@ -0,0 +1,2 @@
+<h1>About Bearded Bear Tracks</h1>
```
Comment (Brit200313) [inline app/views/welcome/about.html.erb]: Hooray!

**Comment by Brit200313** [comment]
@JakeWengroff Looking good! Let's get in the habit of deleting unused files. Once those are removed, you can merge this PR in.

**Comment by JakeWengroff** [comment]
Hi @Brit200313!  Thanks for having a look.  I deleted the files directly on GitHub -- the first time I've done this.  
I will not Merge the Pull Request.

**Pull request merged**